main.go

```diff
@@ -99,6 +97,11 @@ func main() {
 		TimeEncoder: zapcore.RFC3339TimeEncoder,
 	}
 	opts.BindFlags(flag.CommandLine)
+
+	flag.DurationVar(&baseDelay, "rate-limiter-base-delay", 500*time.Millisecond, "The base delay for the rate limiter.")
+	flag.DurationVar(&maxDelay, "rate-limiter-max-delay", 15*time.Minute, "The maximum delay for the rate limiter.")
```

Comment:
nit: ideally would prefer a unit in the name itself.

Reply:
why is `rate-limiter-base-delay-seconds=0.5` or `rate-limiter-base-delay-seconds=1` prefereable over
`rate-limiter-base-delay=500ms` or `rate-limiter-base-delay=1s` ? The latter is a more standard and versatile way of admitting input.